packages/framework/src/client.ts

```diff
@@ -838,3 +848,26 @@ function buildSteps(stateArray: State[]) {
 
   return result;
 }
+
+const sanitizeOutputs = (stepType: string, output: Record<string, unknown>): Record<string, unknown> => {
+  const result: Record<string, unknown> = {};
+
+  for (const [key, value] of Object.entries(output)) {
+    if (skipOutputSanitization(stepType, key)) {
+      result[key] = value;
+    } else {
+      Object.assign(result, sanitizeHtmlInObject({ [key]: value }));
+    }
+  }
+
+  return result;
+};
+
+const skipOutputSanitization = (stepType: string, outputKey: string) => {
+  // Skip sanitization for string values that aren't HTML
+  if (outputKey === 'subject' && stepType === 'email') {
+    return true;
+  }
+
+  return false;
+};
```

Comment:
We need to change this and do the opposite and sanitize only HTML outputs instead, as this could happen on all of the channels.